### ISSUE TITLE
A11y testing

### ISSUE THREAD

**Issue by Akryum** [histoire-dev/histoire#85]
We could leverage [axe](https://github.com/dequelabs/axe-core) to get accessibility testing for stories.

**Comment by alvarosabu** [comment]
Hi, something similar to this addon https://storybook.js.org/addons/@storybook/addon-a11y?

**Comment by alvarosabu** [comment]
@Akryum I will like to take this one, since I need it for my product.

Any wish list?

**Comment by alvarosabu** [comment]
@Akryum Is it possible to create a new tab through the Plugin API? Also how to add the UI inside the tab (for example a list of the accessibility violations and links) 

<img width="389" alt="Screenshot 2022-08-09 at 15 24 46" src="https://user-images.githubusercontent.com/4699008/183658212-0e90405d-355b-4a19-b804-3ea56ac0bc20.png">

**Comment by Akryum** [comment]
That's thing that we would need to add to the Plugin API. Do you confirm you need the following for now?
- [ ] Add a new tab
- [ ] Render a component in it using the bundled Vue (integrate with the UI).

**Comment by alvarosabu** [comment]
Yes, I think those two points will set the bases for plugins that need UI. 

I wonder how we could access the current story document from the API ? With `api.onPreviewStory`? 

I need to investigate if axe only works with a selector or if you can actually pass HTML, not sure

**Comment by Akryum** [comment]
https://github.com/histoire-dev/histoire/discussions/240
